src/@utils/aquarius.ts

```diff
@@ -22,6 +23,11 @@ export function escapeEsReservedCharacters(value: string): string {
   return value.replace(pattern, '\\$1')
 }
 
+export interface ShaclSchemaRes {
```

Comment:
and in general do not type out things already in axios. As with all our other axios usage, this is how you type their responses:

```
type Asset = { example: string }
const response: AxiosResponse<Asset> = await axios.get()
```